instrumentation/github.com/gin-gonic/gin/otelgin/config.go

```diff
@@ -115,3 +120,11 @@ func WithMetricAttributeFn(f MetricAttributeFn) Option {
 		c.MetricAttributeFn = f
 	})
 }
+
+// WithGinMetricAttributeFn specifies a function that extracts additional attributes from the gin.Context
+// and returns them as a slice of attribute.KeyValue.
```

Comment:
```suggestion
// and returns them as a slice of attribute.KeyValue.
//
// If attributes are duplicated between this method and `WithMetricAttributeFn`, the attributes in this method will be used.
```

We should also add the equivalent comment on the `WithMetricAttributeFn` method.

Reply:
Can you also add the equivalent comment to the `WithMetricAttributeFn` method?

Reply:
Yes, I have added